docs/integrations/spryker-glue-api/getting-started-with-apis/getting-started-with-apis.md

```diff
@@ -0,0 +1,141 @@
+- Sorting: Request data to be sorted using parameters like `?sort=attributeName` (ascending) or `?sort=-attributeName` (descending). Sorting parameters can be retrieved using `$glueRequestTransfer->getSortings()`.
+- Filtering: Narrow down results using filter parameters, often structured like `?filter[resourceName.fieldName]=value`. These are accessible via `$glueRequestTransfer->getFilters()`.
+- Sparse Fields: To receive only specific fields of a resource and reduce data transfer, use `?fields[resourceName]=attribute1,attribute2`. This is retrieved using `$glueRequestTransfer->getQueryFields()`.
+- Including Related Resources: Fetch related data in a single request using the include parameter–for example, `?include=concrete-product-image-sets`). The behavior of this included section can be configured in Spryker.  
+
+
+### Glue API Responses
+
+- Status Codes: Standard HTTP status codes indicate the outcome, such as 200 OK, 201 Created, 400 Bad Request, 401 Unauthorized, or 404 Not Found.
+- Response Body: The structure often follows the JSON:API convention.
+  - The resources field in `GlueResponseTransfer` typically contains an array of the primary data objects.
+  - An included section may contain data for related resources if requested via the include parameter.
+  - Responses often include pagination links, such as next, previous, last, first, automatically calculated when using JSON:API.
+  - The overall response structure is defined by GlueResponseTransfer, which includes fields for HTTP status, metadata, content/resources, errors, filters, sortings, and pagination details.
+
+
+### Authentication with Spryker
+
+Glue API primarily uses OAuth 2.0 for securing endpoints.
+
+Client applications send user credentials (username and password) to an authentication endpoint (for example, `/token` for SAPI or a separate `/token` for BAPI using Back Office credentials) to obtain an access token and a refresh token. This access token (Bearer token) must then be included in the Authorization header for subsequent requests to protected Spryker resources. If an invalid, expired, or no token is provided for a protected resource, the API will respond with a 401 Unauthorized status code.
+
+---
+
+## Querying GLUE API Data
+
+The Glue API supports standardized ways to query and manipulate data, especially when using conventions like JSON:API. This makes client interactions predictable and efficient. Key mechanisms are as follows:
+
+- Pagination: Handle large datasets effectively by requesting data in manageable chunks (pages).
+- Sorting: Allow clients to specify the order in which results should be returned.
+- Filters: Enable clients to retrieve subsets of resources based on specific criteria.
+- Sparse Fields: Optimize response payloads by allowing clients to request only the specific data fields they need.
+- To learn more, explore: "Querying Data with Glue API: Standard Parameters (Pagination, Sorting, Filters, Sparse Fields)."
```

Comment:
If you're referencing something, provide a link. I'm not sure where this is supposed to go